### PR TITLE
About page

### PR DESCRIPTION
**Pull request by DavidVorick**
I slightly expanded the index page, and then rewrote  the 'about' page to include a lot more technical detail.

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -39,27 +39,46 @@
...
-							A file will survive even if dozens of hosts drop offline at the same time.
-							Having the file in hundreds of places enables massively parallel downloads, like a torrent with hundreds of seeders.
-							The throughput on Sia should max out most consumer-grade connections.
+							Sia specifically uses erasure coding, primarily Reed-Solomon coding, to add redundancy to the network.
+							Reed-Solomon coding is an M-of-N erasure code with optimal efficiency.
+							Out of N total copies of the data, only M are required to recover the data, and the overhead is N/M.
+							For example, on the Sia network we can have 60 total pieces of data, of which only 20 are required to recover the original, and the overhead is only 3.
+							Using that scheme, even if half of the network goes offline simultaneously, the original data can still be recovered.
+							Once the original datat is recovered, the full redundancy of 60 pieces can be restored.
+							When the network is live, we will use statisical analysis to determine how much redundancy is required.
```
Comment (lukechampine) [inline about.html]: typo

File: about.html [hunk 2]
```diff
@@ -39,27 +39,46 @@
...
+							All data on Sia is encrypted before being uploaded.
+							Nobody can view the data without the encryption key, not even the hosts protecting the data.
+							The user has full control over who gets the key.
+							Twofish is used as the default encryption algorithm.
 							<br><br>
-							Anybody can join the network as a host, and anybody can use the network for storing files.
-							Every client receives the same quality of service, whether they are renting 5 megabytes or 500 terabytes.
-							There is no file discrimination, and there are no corporate deals or favors that occur over the network.
+							Hosts are kept honest by using contracts on a blockchain.
+							When a file is uploaded to a host, a contract is created that says "at time X, the host must have Y data."
```
Comment (lukechampine) [inline about.html]: shouldn't it be "data Y"?

File: about.html [hunk 3]
```diff
@@ -39,27 +39,46 @@
...
+							When uploading, users can select the hosts that best fit their needs.
+							For example, a user in China is likely to select a different set of hosts than a user in America.
+							Most of the selection and optimization is done automatically.
+							<br><br>
+							It's difficult to know exactly where the Sia storage price will end up.
+							Nobody could have predicted that the price of Bitcoin hashing would drop so low - many orders of magnitude below what many people would have initially thought was possible.
+							Today, the standard consumer (non-bulk) price of hard drives is around $35/TB.
+							After redunancy, servers, and electricity, we expect the total cost of storage to be $100/TB, with the components lasting 18 months on average.
+							This results in a monthly cost of $6/TB.
+							The service provided by Sia is likely to be an improvement to the service provided by Amazon S3, which costs at best $27/TB/mo.
```
Comment (lukechampine) [inline about.html]: improvement upon

File: index.html [hunk 4]
```diff
@@ -43,21 +43,32 @@
...
 						Anybody can join as a host and make money.
 						This openness allows Sia to take advantage of all the unused storage available in the world, estimated to be millions upon millions of terabytes.
 						This creates a great opportunity for cheap storage.
 						Reliability is maintained by using redundancy and cryptography.
 						<br><br>
-						Sia is currently just a testnet, with downloads available at the bottom of the page.
+						Sia's decentralized approach to cloud storage results in a global network of computers that can be leveraged to improve upon price, efficiency, security, reliability, latency, and throughput over traditional cloud storage.
+						Many of these advantages come from leveraging the network to create advanced algorithms for storing and retrieving data.
+						Many advantages also result from directly correlating storage and bandwidth with income.
+						We saw a similiar phenomenon with Bitcoin mining, where hashrate is very tightly coorelated to income.
```
Comment (lukechampine) [inline index.html]: correlated

File: index.html [hunk 5]
```diff
@@ -43,21 +43,32 @@
...
+						There is no marketing or customer service overhead for Bitcoin mining.
+						The result was that Bitcoin mining became absurdly efficient, today Bitcoin does more computation than the top 500 supercomputers combined.
```
Comment (lukechampine) [inline index.html]: you want a colon or semicolon here, not a comma.

File: index.html [hunk 6]
```diff
@@ -43,21 +43,32 @@
...
+						Sia brings the same direct coorelation between profit and efficient infrastructure.
```
Comment (lukechampine) [inline index.html]: correlation

File: index.html [hunk 7]
```diff
@@ -43,21 +43,32 @@
...
-						Before ever being uploaded, your data is split into hundreds of pieces and each piece is encrypted.
-						Then, your data is uploaded to dozens of hosts on the network.
-						A blockchain is used to create cryptographic contracts with each service, ensuring that they only get paid if they actually store your data.
-						Out of dozens of hosts, only 1/3 need to remain online for you to recover the data.
-						Even if many of the hosts are unreliable, your files are still safe.
+						Sia is a new decentralized cloud storage platform aimed at giving users control of their data.
+						While uploading, data is split into hundreds of pieces and encrypted client-side.
+						Each piece is then uploaded to a separate host on the network.
+						A blockchain is used to create cryptographic contracts with each service, ensuring that they only get paid if they actually store the data.
+						Out of dozens of hosts, only a fraction need be avilable for the data to be recovered.
```
Comment (lukechampine) [inline index.html]: available

File: index.html [hunk 8]
```diff
@@ -43,21 +43,32 @@
...
+						Sia's decentralized approach to cloud storage results in a global network of computers that can be leveraged to improve upon price, efficiency, security, reliability, latency, and throughput over traditional cloud storage.
+						Many of these advantages come from leveraging the network to create advanced algorithms for storing and retrieving data.
+						Many advantages also result from directly correlating storage and bandwidth with income.
+						We saw a similiar phenomenon with Bitcoin mining, where hashrate is very tightly coorelated to income.
+						There is no marketing or customer service overhead for Bitcoin mining.
+						The result was that Bitcoin mining became absurdly efficient, today Bitcoin does more computation than the top 500 supercomputers combined.
+						Sia brings the same direct coorelation between profit and efficient infrastructure.
+						Storage that is plugged into the network will make money for the owner, with no marketing or customer service overhead.
+						We believe that this, combined with intelligent algorithms for levaraging such a network, will create the best cloud storage platform the world has ever seen.
```
Comment (lukechampine) [inline index.html]: leveraging

**Comment by DavidVorick** [comment]
man I dropped this ball pretty hard.

Running spellcheck...

**Comment by DavidVorick** [comment]
better now.

Also picked up some spelling errors on the other pages.

**Pull request merged**